.github/workflows/linux.yml

```diff
@@ -76,13 +82,13 @@ jobs:
         run: |
           export _JAVA_OPTIONS="-Xmx4G"
           export DOCKER_BUILDKIT=1
-          ./gradlew --parallel packageJdk${{ matrix.distro }} checkJdk${{ matrix.distro }} -PPRODUCT=${{ matrix.product.name }} -PPRODUCT_VERSION=${{ matrix.product.version }} --stacktrace
+          ./gradlew --parallel package${{ matrix.image_type }}${{ matrix.distro }} check${{ matrix.image_type }}${{ matrix.distro }} -PPRODUCT=${{ matrix.product.name }} -PPRODUCT_VERSION=${{ matrix.product.version }} --stacktrace
 
       - name: Relocate test results
         if: always() # always run even if the previous step fails
         run: |
           mkdir ${{ matrix.product.version }}
-          mv jdk ${{ matrix.product.version }}
+          mv ${{ matrix.image_type.lowercase }} ${{ matrix.product.version }}
```

Comment:
```suggestion
          mv $(echo ${{ matrix.image_type }} | tr 'A-Z' 'a-z') ${{ matrix.product.version }}
```

Reply:
Well, this was already implemented exactly this way in my latest (forced) push yesterday. For some reason, GH didn't make this part of the PR? 

However, I have refactored my refactoring anyways to make it most compatible with the previous state. Wait for it ... It's just testing. Then I will force push to the PR again.